tc/sgx/trusted_worker_manager/enclave/enclave.edl

```diff
@@ -20,7 +20,7 @@ enclave {
     from "signup.edl" import *;
     from "signup_kme.edl" import *;
     from "signup_wpe.edl" import *;
-    from "workload.edl" import *;
+    from "work_order.edl" import *;
```

Comment:
Because the APIs defined in the edl file are specific to handling work order requests and they don't directly deal with workloads. Hence, the renaming.